twisted/test/test_compat.py

```diff
@@ -315,6 +317,26 @@ def test_python3(self):
 
 
 
+class PYPYTest(unittest.SynchronousTestCase):
+    """
+    Identification of PyPy.
+    """
+
+    def test_pypy(self):
+        """
+        On PyPy, C{_PYPY} is True.
+        """
+        if 'PyPy' in sys.version:
+            self.assertTrue(_PYPY)
+
```

Comment:
I agree, but I chose to follow `PY3Tests`' example.  Should we have a new ticket for that?